starport/services/network/request.go

```diff
@@ -0,0 +1,56 @@
+package network
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	launchtypes "github.com/tendermint/spn/x/launch/types"
+	"github.com/tendermint/starport/starport/pkg/events"
+	"github.com/tendermint/starport/starport/services/network/networkchain"
+)
+
+// Reviewal keeps a request's reviewal.
+type Reviewal struct {
+	RequestID  uint64
+	IsApproved bool
+}
+
+// ApproveRequest returns approval for a request with id.
+func ApproveRequest(requestID uint64) Reviewal {
+	return Reviewal{
+		RequestID:  requestID,
+		IsApproved: true,
+	}
+}
+
+// RejectRequest returns rejection for a request with id.
+func RejectRequest(requestID uint64) Reviewal {
+	return Reviewal{
+		RequestID:  requestID,
+		IsApproved: false,
+	}
+}
+
+// SubmitRequest submits reviewals for proposals in batch for chain.
+func (n Network) SubmitRequest(launchID uint64, reviewal ...Reviewal) error {
+	n.ev.Send(events.New(events.StatusOngoing, "Submitting requests..."))
```

Comment:
We also can add a submitted event.

Reply:
maybe it can cause redundancy in the logs message. We already have this one

https://github.com/tendermint/starport/pull/1809/files#diff-15a72430f6e3b4fbfbb089ebdea84d24a74cb6aad21b0854691cdf1abe51a483R68

Reply:
I see, in some funcs we print the Ongoing and Done status, and in some only Ongoing. This confuses me a bit and kinda hard to track in code. Maybe we can figure out this later, set a certain rule about it or refactor events/logs after merging all PRs.

Reply:
Agree. This method is reusable for the approve and reject requests, which is why I put the print outside the method. We can call the event system from the cmd instead use `fmt.Print`